roles/comanage-registry/tasks/main.yml

```diff
@@ -7,9 +7,55 @@
     - "{{ ansible_distribution }}-{{ ansible_distribution_major_version}}.yml"
     - "{{ ansible_distribution }}.yml"
     - "{{ ansible_os_family }}.yml"
+  tags:
+  - always
 
-# Include OS-independent installation tasks
-- include: install.yml
+# Tasks deploying COmanage framework. There is none conditional task.
+- name: Include OS-independent installation tasks
+  include_tasks:
+    file: install.yml
+    apply:
+      tags:
+      - install
+  tags:
+  - install
 
-# Include OS-independent configuration tasks
-- include: configure.yml
+# Tasks required for basic Framework functionality
+- name: Include OS-independent setup tasks
+  include_tasks:
+    file: setup.yml
+    apply:
+      tags:
+      - setup
+  tags:
+  - setup
+
+  # Tasks required for basic Framework functionality
+- name: Include OS-independent initialize tasks
+  include_tasks:
+    file: initialize.yml
+    apply:
+      tags:
+        - initialize
+  tags:
+    - initialize
+
+# Tasks that are Deployment Specific
+- name: Include OS-independent configuration tasks
+  include_tasks:
+    file: configure.yml
+    apply:
+      tags:
+      - configure
+  tags:
+  - configure
+
+# Tasks adding/copying/replacing files
+- name: Include OS-independent Update/Copy tasks
+  include_tasks:
+    file: static_files.yml
+    apply:
+      tags:
+      - staticfiles
+  tags:
+  - staticfiles
```

Comment:
Aren't these considered configuration tasks? I think it would make more sense to include static_files.yml from configure.yml and use a tag like `comanage-registry:config:staticfiles`

Reply:
Done